pkg/authorization/authorizer/cache/authorizer.go

```diff
@@ -91,37 +90,6 @@ func (c *CacheAuthorizer) Authorize(a kauthorizer.Attributes) (allowed bool, rea
 	return allowed, reason, err
 }
 
-func (c *CacheAuthorizer) GetAllowedSubjects(attributes kauthorizer.Attributes) (sets.String, sets.String, error) {
```

Comment:
I know you've split the interface in 2. Does the code that is using the `CacheAuthorizer` not ever call `GetAllowedSubjects`?